youtube_dl/downloader/dash.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import unicode_literals
+from .common import FileDownloader
+from ..compat import compat_urllib_request
+
+import re
```

Comment:
The import order convention we try to follow is:
1. __future__ imports
2. standard library imports
3. internal imports (aka: all relative imports)